docs/technical_reference/flowsheets/mvc.rst

```diff
@@ -0,0 +1,125 @@
+concentrated brine and distillate pass through heat exchangers to preheat the incoming feedwater before exiting the
+system. The flowsheet relies on the following key assumptions:
+
+   * supports steady-state only
+   * property package(s) supporting liquid and vapors is provided
+
+.. figure:: ../../_static/flowsheets/mvc.png
+    :width: 600
+    :align: center
+
+    Figure 1. MVC flowsheet
+
+Documentation for each of the WaterTAP unit models can be found below.
+    * `Pressure Changer <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/pressure_changer.html>`_
+    * `Evaporator <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/mvc.html>`_
+    * `Compressor <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/mvc.html>`_
+    * `Condenser <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/mvc.html>`_
+
+Documentation for each of the IDAES unit models can be found below.
+    * `Feed <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/feed.html>`_
+    * `Heat Exchanger <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/heat_exchanger.html>`_
+    * `Separator <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/separator.html>`_
+    * `Product <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/product.html>`_
+    * `Mixer <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/mixer.html>`_
+    * `Translator <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/translator.html>`_
+
+Documentation for each of the property models can be found below.
+    * `Water <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/water.html>`_
+    * `Seawater <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/seawater.html>`_
+
+Documentation for the costing relationships can be found below.
+    * `WaterTAP Costing Package <https://watertap.readthedocs.io/en/latest/technical_reference/costing/watertap_costing.html>`_
+
+The objective function is to minimize the levelized cost of water, which can be represented by the following equation
+where :math:`Q` represents volumetric flow, :math:`f_{crf}` represents capital recovery factor
+:math:`C_{cap,tot}` represents total capital cost, :math:`C_{op,tot}` represents total operating cost, and
+:math:`f_{util}` represents the utilization factor:
+
+    .. math::
+
+        LCOW_{Q} = \frac{f_{crf}   C_{cap,tot} + C_{op,tot}}{f_{util} Q}
+
+Degrees of Freedom
+------------------
+The following variables are initially specified for simulating the MVC flowsheet (i.e., degrees of freedom = 0):
```

Comment:
I don't believe we need to list all the variables fixed for initialization purposes (DOF=0). The main goal of this flowsheet is to find the optimum values for the evaporator area, brine and distillate heat exchanger areas, evaporator temperature, and compressor pressure ratio. I suggest removing the DOF=0, and the mentioned variables from the list

Reply:
I have made suggestions accordingly

Reply:
Done